bertopic/representation/_utils.py

```diff
@@ -57,6 +58,21 @@ def decode(self, doc_chunks):
         return truncated_document
     return document
 
+def validate_truncate_document_parameters(tokenizer, doc_length) -> Union[None, ValueError]:
+    """validates parameters that are used in the function `truncate_document`"""
+    if tokenizer is None and doc_length is not None:
+        raise ValueError(
+            "Please select from one of the valid options for the `tokenizer` parameter: \n"
+            "{'char', 'whitespace', 'vectorizer'} \n"
+            "If `tokenizer` is of type callable ensure it has methods to encode and decode a document \n"
+        )
+    elif tokenizer is not None and doc_length is None:
+        raise ValueError(
+            "If `tokenizer` is provided, `doc_length` of type int must be provided as well."
+        )
+    else:
+        pass
```

Comment:
I believe this isn't necessary right?

Reply:
No it is not necessary but I included it to be verbose in terms of the possible edgecase. That being said, happy to remove it.

Reply:
Let's remove it and I'll make sure to merge it after that! Other than that, it looks great!